var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -219,6 +219,7 @@ def std_when(values):
     patch("cxx17.patch", when="@10.3 cxxstd=17")
     patch("cxx17_geant4_10_0.patch", level=1, when="@10.4.0 cxxstd=17")
     patch("geant4-10.4.3-cxx17-removed-features.patch", level=1, when="@10.4.3 cxxstd=17")
+    patch("geant4-10.7-cxx20-g3tog4.patch", when="@:10.7 cxxstd=20")
```

Comment:
Can you add a brief comment above the `patch` directive explaining what is the effect of the patch (it helps when reading the package)?

Reply:
Should've known better to do that. Done!